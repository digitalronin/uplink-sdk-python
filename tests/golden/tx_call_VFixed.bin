00000000: 03 EA 09 F9 55 4A E9 80  4C 33 00 AB CA 52 3A 63  ....UJ..L3...R:c
00000010: 6A 53 AF 81 F6 39 AD 6E  BA 17 0E 05 18 F0 3F E2  jS...9.n......?.
00000020: 9E 29 00 00 00 00 00 00  00 03 67 65 74 00 00 00  .)........get...
00000030: 00 00 00 00 01 03 02 01  00 02 97 0C              ............