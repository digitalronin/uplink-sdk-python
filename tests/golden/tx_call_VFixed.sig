r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 42376926774667238710521486878700670091080731708364783778300212657338688052376
