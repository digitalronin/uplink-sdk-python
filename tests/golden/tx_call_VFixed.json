{
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "header": {
        "tag": "TxContract", 
        "contents": {
            "tag": "Call", 
            "contents": {
                "args": [
                    {
                        "tag": "VFixed", 
                        "precision": 3, 
                        "contents": {
                            "tag": "Fixed3", 
                            "contents": 3.223
                        }
                    }
                ], 
                "method": "get", 
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve"
            }
        }
    }, 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE04NDEzNDAyODMwNzU3ODQ2NzE5MTE1NjQ4MTQyODA5ODEyOTI2NjAwMDQ5NzU4Njk5NzI0NDU1MzM5Nzg5ODEzODc3MDAxNDEyMTg4NA=="
}